src/Components/Web/src/Forms/InputFile/IBrowserFile.cs

```diff
@@ -38,7 +38,20 @@ public interface IBrowserFile
         /// <summary>
         /// Opens the stream for reading the uploaded file.
         /// </summary>
+        /// <param name="maxAllowedSize">
+        /// The maximum size that can be read from the Stream. Defaults to 500kb.
+        /// <para>
+        /// Calling <see cref="OpenReadStream(long, CancellationToken)"/>
+        /// will throw if the uploaded file size, as specified by <see cref="Size"/> is larger than
```

Comment:
```suggestion
        /// will throw if the selected file's size, as specified by <see cref="Size"/> is larger than
```

Reply:
I recommend we avoid using the term "upload" anywhere, since (1) the file hasn't actually been transferred by the time this method runs, and (2) in the wasm case, the file doesn't necessarily leave the user's computer at all.

People have been confused into thinking this is about HTTP uploads, but it's not - it's about making file data readable by .NET code.